src/datasets/batch_view/main_summary/intro.md

```diff
@@ -18,4 +22,4 @@ limit to a short submission date range.
 #### Accessing the Data
 
 The `main_summary` table is accessible through re:dash.
-Here's an [example query](https://sql.telemetry.mozilla.org/queries/4201/source).
\ No newline at end of file
+Here's an [example query](https://sql.telemetry.mozilla.org/queries/4201/source).
```

Comment:
Doesn't look like any change on this line?

Reply:
Maybe it's whitespace? I'll try to figure that out.

Reply:
A local `git diff` shows me that there was no newline at the end of the file previously, so I think that's what's up here. But I thought I had seen GitHub display that explicitly in other scenarios.